bitriseclient/models.go

```diff
@@ -0,0 +1,155 @@
+package bitriseclient
```

Comment:
i would may split the code by request groups:

- profiles.go
- certificates.go

move the models to the request group's file, they belongs together